docs/UserGuide.md

```diff
@@ -141,6 +141,11 @@ The fields are:
 - `p/PHONE` is the phone number of the Company.
 - `a/ADDRESS` is the address of the Company.
 
+Note:
+- Connectify only supports alphanumeric characters and spaces for names.
+- There are no restrictions on the input for the website field as it is for the user's personal reference. This means that websites can be successfully inputted even without the `www` prefix.
+- All fields are case-sensitive. This means that `Apple Inc` and `apple inc` are considered different names.
```

Comment:
Thanks Gerald! This addresses some of the concerns raised by testers previously.